docs/errors/main.md

```diff
@@ -1,30 +1,30 @@
-# Common Errors
-In this section we explain a number of common error codes that users experience in the real world.
+# Genel Hatalar
+Bu bölümde kullanıcıların gerçek hayatta deneyimledikleri birkaç genel hata kodunu açıklayacağız.
 
 ## TS2304
-Samples:
-> `Cannot find name ga`
+Örnek:
+> `Cannot find name ga (ga ismi bulunamıyor)`
 
-You are probably using a third party library (e.g. google analytics) and don't have it `declare`d. TypeScript tries to save you from *spelling mistakes* and *using variables without declaring them* so you need to be explicit on anything that is *available at runtime* because of you including some external library ([more on how to fix it][ambient]).
+Muhtemelen `declare (deklare)` etmediğiniz üçüncü parti bir kütüphane (örneğin google analytics) kullanıyorsunuz. TypeScript sizi *yazım hatalarından* ve *değişkenleri deklare etmeden kullanmanızdan* korumaya çalışır, yani *çalışma zamanında varolan* harhangi bir şeye açık olmalısınız çünkü bazı dış kütüphaneleri dahil ediyorsunuz ([nasıl düzeltileceğine dair dahası][ambient]).
```

Comment:
Bence şu şekil daha doğru linkler için;
([nasıl düzeltileceğine dair daha fazla bilgi][ambient])